app/models/publication.rb

```diff
@@ -65,7 +65,11 @@ def self.open_access_statuses
   end
 
   def self.oa_workflow_states
-    ['automatic DOI verification pending']
+    ['automatic DOI verification pending', 'automatic permissions check pending', 'error during permissions check']
+  end
+
+  def self.preferred_versions
+    ['acceptedVersion', 'publishedVersion']
   end
```

Comment:
We should use the values stored in the translation ymls to set the preferred versions values.  Like how we set valid versions in the oab_permissions_client: https://github.com/psu-libraries/researcher-metadata/blob/fd12910011be99c5931ad208eaa8f7724193ef87/app/services/oab_permissions_client.rb#L7-L8 .